### ISSUE TITLE
incomplete list of vlans

### ISSUE THREAD

**Issue by sbellamine** [GlobalNOC/VCE#167]
<port name="ethernet 3/1" owner="admin">
      <tags start="100" end="199" workgroup="admin"/>	
      <tags start="205" end="205" workgroup="admin"/>

Issue: vlans on the second line are not available via the front end interface.
In this case, vlan 205 is not shown in the GUI as an available vlan.

Thank you

**Comment by ajragusa** [comment]
Could you please provide an example configuration.  I'm not sure what you mean by vlans on the second line are not available.  Please send the relevant part of the config so we can reproduce and then fix.

Thanks
A.J.

**Comment by sbellamine** [comment]
Please see relevant part of the config and a more detailed explanation of the issue:

  <port name="ethernet 3/1" owner="admin">
      <tags start="100" end="199" workgroup="admin"/>	
      <tags start="205" end="205" workgroup="admin"/>
      <tags start="201" end="201" workgroup="testing"/>

- Logged in as user admin
- Under “Ports”, I can see that I have access to vlans 100-199, 205 ( as expected )
- But under vlans, things do not work as expected:
-- Go to “vlans” tab
-- Select Create
-- Add endpoint Eth3/1
-- Under vlan, vlan id “205” is not available. Only available vlans in the range 100-199 are. 
So this entry   <tags start="205" end="205" workgroup="admin"/> is not available under the vlans tab.

**Comment by jonstout** [comment]
When provisioning, the second vlan range defined under a port is not available for provisioning. Verify that when multiple ranges are added, all vlan are available for provisioning in the gui.

```
<tags start="100" end="199"/>
<tags start="200" end="300"> <!-- this range will not be able to provision in this range -->
```

**Comment by absheth** [comment]
Fixed in #227
